### PR TITLE
Added experimental mixpanel integration

### PR DESCRIPTION
**Pull request by nickoneill**
For #256, last thing to do before 1.5.3 goes out.

### CODE REVIEW

File: FiveCalls/FiveCalls/AppDelegate.swift [hunk 1]
```diff
@@ -41,6 +42,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         }
 
         oneSignalStartup(launchOptions: launchOptions)
+        Mixpanel.sharedInstance(withToken: "776fce75f7e3ddfbb13b615dcb94ff95")
```
Comment (subdigital) [inline FiveCalls/FiveCalls/AppDelegate.swift]: Do we already have a place to store this stuff? I don't like committing tokens in the source when I can avoid it.

File: FiveCalls/FiveCalls/IssueDetailViewController.swift [hunk 2]
```diff
@@ -27,6 +28,8 @@ class IssueDetailViewController : UIViewController, IssueShareable {
         tableView.estimatedRowHeight = 100
         tableView.rowHeight = UITableViewAutomaticDimension
         NotificationCenter.default.addObserver(self, selector: #selector(madeCall), name: .callMade, object: nil)
+
+        Mixpanel.sharedInstance()?.track("Topic", properties: ["IssueID": self.issue.id])
```
Comment (subdigital) [inline FiveCalls/FiveCalls/IssueDetailViewController.swift]: What would you say to an extension on `UIViewController` like this:

```swift
extension UIViewController {
    func trackEvent(_ name: String, properties: [String:String]) {
        Mixpanel.sharedInstance()?.track(...)
    }
}
```

Comment (subdigital) [reply FiveCalls/FiveCalls/IssueDetailViewController.swift]: This way we decouple the actual type (and service) used to track these events and cleans up the callsite a bit.

File: FiveCalls/FiveCalls/IssuesViewController.swift [hunk 3]
```diff
@@ -56,7 +57,12 @@ class IssuesViewController : UITableViewController {
         self.registerForPreviewing(with: self, sourceView: tableView)
 
         Answers.logCustomEvent(withName:"Screen: Issues List")
-        
+        if self.shouldShowAllIssues {
+            Mixpanel.sharedInstance()?.track("More")
+        } else {
+            Mixpanel.sharedInstance()?.track("Home")
+        }
```
Comment (subdigital) [inline FiveCalls/FiveCalls/IssuesViewController.swift]: With the above change, we could extract this into a single call:

```swift
var analyticsEvent: String {
    if shouldShowAllIssues { 
        return "More"
    } else {
        return "Home"
    }
}

...

trackEvent(analyticsEvent)

```

File: FiveCalls/FiveCalls/IssueDetailViewController.swift (hunk 2 above)
Comment (nickoneill) [reply FiveCalls/FiveCalls/IssueDetailViewController.swift]: 👍

File: FiveCalls/FiveCalls/AppDelegate.swift (hunk 1 above)
Comment (nickoneill) [reply FiveCalls/FiveCalls/AppDelegate.swift]: Info.plist seems like what we've been using. I'll pop it in there.

**Comment by nickoneill** [comment]
Updated!

File: FiveCalls/FiveCalls/IssueDetailViewController.swift [hunk 4]
```diff
@@ -10,6 +10,7 @@ import UIKit
 import CoreLocation
 import Crashlytics
 import Down
+import Mixpanel
```
Comment (subdigital) [inline FiveCalls/FiveCalls/IssueDetailViewController.swift]: don’t need this in your view controllers anymore, since you’re calling an extension method.

**Comment by subdigital** [comment]
Otherwise looks good

**Comment by nickoneill** [comment]
Updated and rebased, will merge after the checks complete.

**Pull request merged**